public/docs/_examples/toh-6/ts/app/hero-detail.component.ts

```diff
@@ -29,11 +31,11 @@ export class HeroDetailComponent implements OnInit {
   // #docregion save
   save(): void {
     this.heroService.update(this.hero)
-      .then(this.goBack);
+      .then(() => this.goBack());
```

Comment:
Why this change? You are replacing a function g by an anonymous function which does nothing but call g. I.e., g and `() => g()` are equivalent, though the former is shorter.